meta-genivi-dev/meta-ivi/recipes-extended/node-state-manager/node-state-manager_2.0.0.bbappend

```diff
@@ -1,5 +1,13 @@
-EXTRA_OECONF_append=" --with-nsmc=NodeStateMachineTest "
+NSMC="SimpleNodeStateMachine"
+EXTRA_OECONF_append=" --with-nsmc=${NSMC} "
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-do_install_append() {
-    install -m 755 ${B}/NodeStateMachineTest/NodeStateTest ${D}${bindir}/NodeStateTest
+SRC_URI_append = "file://0001-Build-SimpleNodeStateMachine.patch "
+
+do_configure_prepend() {
+    git clone https://github.com/GENIVI/simple-node-state-machine.git ${WORKDIR}/git/${NSMC}
+    cd ${NSMC}
+    git checkout b4de5dee4d70a223eef37bcbb5f8536560b0e243
+    touch README NEWS AUTHORS ChangeLog
```

Comment:
They where not part of the package and autotools failed because of it. I've since added them to the package so can remove the line now.